cmd/miniooni/main.go

```diff
@@ -270,6 +272,8 @@ func main() {
 		experiment = web_connectivity.NewExperiment(sess, web_connectivity.Config{})
 	} else if name == "whatsapp" {
 		experiment = whatsapp.NewExperiment(sess, whatsapp.Config{})
+	} else if name == "example" {
+		experiment = example.NewExperiment(sess, example.Config{10 * time.Second})
```

Comment:
I suggest to use `1` second as the timeout because `10` seconds is probably annoying :-D

Reply:
Maybe `2` second?  (`time.Second` == `1s`) 
Although this is an example. :D

Reply:
Sure, also two seconds is fine